content/en/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions.md

```diff
@@ -518,15 +518,19 @@ apiVersion: "stable.example.com/v1"
 kind: CronTab
 metadata:
   finalizers:
-  - finalizer.stable.example.com
+  - stable.example.com/finalizer
 ```
 
-Finalizers are arbitrary string values, that when present ensure that a hard delete
-of a resource is not possible while they exist.
+Custom finalizers consist of a namespace and a name, separated by a forward slash.
```

Comment:
Since 'namespace' is a special thing in Kubernetes, I'd suggest we avoid using 'namespace' here. What we call a "namespace" here is actually a domain name, and that is all.

Reply:
True, I didn't consider that. I've changed it to be domain name, forward slash and name now.